src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
+        storage.put(transaction.getFruit(), storage.getOrDefault(transaction.getFruit(), 0) + transaction.getQuantity());
+    }
```

Comment:
According to the checklist, we should avoid using if-else constructions and switch-cases when handling operations. It is recommended to use the Strategy pattern instead. Ensure that you have a map of strategies in your service that chooses the correct strategy for each operation.